doc/openshift-logging-rhocp_4.7+.adoc

```diff
@@ -0,0 +1,126 @@
+  outputs:
+  pipelines:
+    - name: parse-liberty-json
+      inputRefs:
+        - liberty-logs
+      outputRefs:
+        - default
+      parse: json
+----
+
+The YAML above sets up a new ClusterLogForwarder pipeline `parse-liberty-json`. This pipeline takes in an input of `liberty_logs`, all the container logs from the `liberty-app` namespace, and outputs them to the OpenShift cluster's default Elasticsearch log store. The `parse: json` definition enables the JSON log parsing.
+
+The configured `outputDefaults.elasticsearch.structuredTypeKey` parameter will build a unique index for the container logs by prepending `app-` to the value of the `logFormat` label in the container. Previously, the label `logFormat: liberty` was added to the service of your OpenLibertyApplication, so the logs forwarded to the Elasticsearch default log store can be found under an index pattern of `app-liberty-*`. If no `logFormat` label exists in your application container, the `outputDefaults.elasticsearch.structuredTypeName` parameter provides a fallback index name.
+
+Deploy the Cluster Log Forwarder instance with the following command:
+
+[source,sh]
+----
+[root@rhel7-ocp ~]# oc create -f cluster-logging-forwarder.yaml
+----
+
+For more information on parsing JSON logs in RHOCP, see the the OpenShift guide link:++https://docs.openshift.com/container-platform/4.8/logging/cluster-logging-enabling-json-logging.html++[Enabling JSON logging].
+
+## View application logs in Kibana
+
+In cases where the application server provides the option, output application logs in JSON format. This lets you fully take advantage of Kibana's dashboard functions. Kibana is then able to process the data from each individual field of the JSON object to create customized visualizations for that field.
+
+See the Kibana dashboard page by using the routes URL <https://kibana-openshift-logging.apps.host.kabanero.com>. Log in using your Kubernetes user and password. The browser will redirect you to Kibana's **Create index pattern** page under **Management**. Create a new index pattern **app-liberty-*** (that was defined above) to select all the Elasticsearch indices used for your Liberty application logs. Navigate to the **Discover** page to view the application logs generated by the deployed application.
+
+image::images/app-logging-ocp-index-pattern-4.7.png[Create new index pattern in Kibana 7]
+
+image::images/app-logging-ocp-discover-4.7.png[Kibana 7 page with the application log entries]
+
+Expand an individual log entry to see the `structured.*` formatted individual fields, parsed and copied out of the nested JSON log entry.
+
+image::images/app-logging-ocp-expand-entry.png[Kibana 7 page with the application log entries]
+
+To import the dashboard and its associated objects, navigate back to the **Management** page and click **Saved Objects**. Click **Import** and select the dashboard file. When prompted, click the **Yes, overwrite all** option
```

Comment:
We need to mention where to get these dashboard files, as the line above seems not to flow from the preceding text.

same comment also applies to the 4.2-4.6 doc.